authors/apps/articles/models.py

```diff
@@ -0,0 +1,20 @@
+from django.db import models
+from authors.apps.authentication.models import User
+
+from cloudinary.models import CloudinaryField
+
+
+class Article(models.Model):
+    """
+    This is the Article model that is used to handle CRUD on articles
+    """
+    title = models.CharField(max_length=200, blank=False)
+    body = models.TextField(blank=False)
+    description = models.CharField(max_length=50, blank=False)
```

Comment:
I think an article should be able to be saved without the title, body or description. The can be left blank. Remember that when the user starts typing, on the frontend we will make requests to save the article.

When it’s about to be published, then we can require that title and body is present, but not necessarily the description